prowler/providers/aws/services/datasync/datasync_service.py

```diff
@@ -0,0 +1,120 @@
+from typing import Dict, List, Optional
+
+from botocore.exceptions import ClientError
+from pydantic import BaseModel, Field
+
+from prowler.lib.logger import logger
+from prowler.lib.scan_filters.scan_filters import is_resource_filtered
+from prowler.providers.aws.lib.service.service import AWSService
+
+
+class DataSync(AWSService):
+    """AWS DataSync service class to list tasks, describe them, and list their tags."""
+
+    def __init__(self, provider):
+        """Initialize the DataSync service.
+
+        Args:
+            provider: The AWS provider instance.
+        """
+
+        super().__init__(__class__.__name__, provider)
+        self.tasks = []
```

Comment:
I think this should be a dict, with the ARN as key. [Doc Ref](https://docs.prowler.com/projects/prowler-open-source/en/latest/developer-guide/services/#service-objects)

Reply:
Good catch @puchy22 !